examples/architectures/VGG8.py

```diff
@@ -37,7 +35,8 @@ def _weight_variable(shape, stddev=0.01):
       stddev: standard deviation of a truncated Gaussian
     """
     initializer = tf.truncated_normal_initializer(stddev=stddev)
-    return tf.get_variable(name='weights', shape=shape, initializer=initializer)
+    return tf.get_variable(name='weights', shape=shape,
+                           initializer=initializer)
```

Comment:
This makes me happy :-) (And it shows another reason to have 80 character lines)